x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Count.java

```diff
@@ -58,7 +58,9 @@ public class Count extends AggregateFunction implements ToAggregator, SurrogateE
             ),
             @Example(
                 description = "To count the same stream of data based on two different expressions "
```

Comment:
I don't understand the `based on two different expressions`. Which 2 different expressions are in that example?
Everything else you added helps a lot btw

Reply:
`ROW n=1
| STATS COUNT(n > 0 OR NULL), COUNT(n < 0 OR NULL)`
is the example added here (so gt, lt).

Reply:
Ah, I thought it was expressions in the same function. I need some extra coffee